### PR TITLE
Fix forbidden code point error (zh_tw)

### PR DESCRIPTION
**Pull request by ismailarilik**
I got this error while running `bundle exec rake check:markup` test command:

```sh
* zh_tw/news/2019/04/23/move-to-git-from-svn/index.html
ERROR; line 111: Forbidden code point U+0008.
```

I have fixed it by looking at examples on other lines.
I think this fix is right and have validated that the test command has passed.
I don't know this language so if there is a mistake, please fix it or warn me and let me fix.

### CODE REVIEW

File: zh_tw/news/_posts/2019-04-23-move-to-git-from-svn.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ lang: zh_tw
 
 * 在 cgit 上，我們不使用 Topic 分支。
 * GitHub 存放區將持續保持鏡像，我們不使用「Merge pull request」功能。
-* ruby_2_4、ruby_2_5 和 ruby_2_6 分支仍使用 SVN，在 cgit 上我們將不會推送任何東西給這些分支。
+* ruby_2_4，ruby_2_5 和 ruby_2_6 分支仍使用 SVN，在 cgit 上我們將不會推送任何東西給這些分支。
```
Comment (bugtender) [inline zh_tw/news/_posts/2019-04-23-move-to-git-from-svn.md]:
```suggestion
* ruby_2_4、ruby_2_5 和 ruby_2_6 分支仍使用 SVN，在 cgit 上我們將不會推送任何東西給這些分支。
```

Comment (bugtender) [reply zh_tw/news/_posts/2019-04-23-move-to-git-from-svn.md]: Sorry, it's my bad. Looks like I push an invisible character before the `、` again.
Like you found in #1983 last time.

But it's a little bit different between `、` and `，` in Chinese.
Could you just delete the invisible character and keep the `、`? Thanks!!

**Comment by ismailarilik** [comment]
I have changed the separator with original one. You can avoid invisible characters by running `bundle exec rake check:markup` command. It outputs this kind of errors. It takes too long to complete but after huge changes, it may be run.

**Comment by JuanitoFatas** [comment]
Thanks @ismailarilik!

**Pull request merged**